2048game/Mmove_2048.py

```diff
@@ -0,0 +1,90 @@
+import random
+
+def move_row_left(row) :
+    """Moves all tiles to the left according to the rules"""
+    taille = len(row)
+    hasBeenModified = [0] * taille 
+    """This list tells us if a tile has already merged during this move (and thus must not be moved again)"""
+    for position in range(taille) :
+        leftmostTileValue,leftmostTilePosition = row_check_tile_direction(row,"left",position) #On récupère la valeur de la première tuile rencontrée
```

Comment:
snake case is more commonly used in python

Reply:
Done.